lib/va_profile/military_personnel/service.rb

```diff
@@ -52,6 +54,37 @@ def get_service_history
         handle_error(e)
       end
 
+      # GET's a user's DoD service summary from the VAProfile API
+      # If a user is not found in VAProfile, an empty DodServiceSummaryResponse with a 404 status will be returned
+      # @return [VAProfile::MilitaryPersonnel::DodServiceSummaryResponse] response wrapper around a
+      # dod_service_summary object
+      def get_dod_service_summary
+        with_monitoring do
+          edipi_present!
+
+          response = perform(:post, identity_path, VAProfile::Models::DodServiceSummary.in_json)
+
+          DodServiceSummaryResponse.from(response)
+        end
+      rescue Common::Client::Errors::ClientError => e
+        error_status = e.status
+        if error_status == 404
+          log_exception_to_sentry(
+            e, { edipi: @user.edipi }, { va_profile: :dod_service_summary_not_found }, :warning
+          )
```

Comment:
We're trying to phase-out our use of Sentry. Is there a reason you need to log to Sentry here or could you do a plain Rails.logger instead? here's some info about logging in vets-api: https://depo-platform-documentation.scrollhelp.site/developer-docs/logging-in-vets-api

(Note: if you change this, you'll also need to update the spec)